FindChannelFunction/src/main/java/no/unit/nva/channel/exception/NoResultsFoundException.java

```diff
@@ -0,0 +1,8 @@
+package no.unit.nva.channel.exception;
+
+public class NoResultsFoundException extends Exception {
```

Comment:
I got the impression that we might be using this for two distinct classes of error